backend/models/viewpoint.js

```diff
@@ -252,51 +276,27 @@ const createViewpoint = async (account, model, collName, routePrefix, hostId, vp
 				systemLogger.logError("Resize failed as screenshot is not a valid png, no thumbnail will be generated", {
 					account,
 					model,
-					type: this.collName,
+					type: collName,
 					id: hostId,
 					err
 				});
 			});
 		}
 
-		await setExternalScreenshotRef(viewpoint, account, model, collName);
+		if (collName) {
+			await Viewpoint.setExternalScreenshotRef(viewpoint, account, model, collName);
+		}
 	}
 
-	return clean(routePrefix, viewpoint, false);
+	return Viewpoint.cleanViewpoint(routePrefix, viewpoint, false);
 };
 
-const setExternalScreenshotRef = async function(viewpoint, account, model, collName) {
+Viewpoint.setExternalScreenshotRef = async function(viewpoint, account, model, collName) {
```

Comment:
is this used outside of this file now? 

how come we made it public?

Reply:
@carmenfan I noticed it's being used by `models/bcf`. Its usage wasn't added in this branch, but I thought I'd fix it since I noticed it.

See here: https://github.com/3drepo/3drepo.io/blob/master/backend/models/bcf.js#L928